emoji-copy@felipeftn/prefs.js

```diff
@@ -1,82 +1,73 @@
-const { GLib, GObject, Gio, Gtk, GdkPixbuf } = imports.gi;
-
-const Gettext = imports.gettext.domain('emoji-copy');
-const _ = Gettext.gettext;
-
-const ExtensionUtils = imports.misc.extensionUtils;
-const Me = ExtensionUtils.getCurrentExtension();
-
-function init() {
-    ExtensionUtils.initTranslations();
-}
-
-const EmojiCopySettingsWidget = new GObject.Class({
-    Name: 'EmojiCopy.Prefs.Widget',
-    GTypeName: 'EmojiCopyPrefsWidget',
+import * as Gtk from 'gi://Gtk';
+import * as GObject from 'gi://GObject';
+import { ExtensionUtils, gettext as _ } from 'resource:///org/gnome/Shell/Extensions/js/extensions.js';
```

Comment:
This line should [probably](https://gjs.guide/extensions/upgrading/gnome-shell-45.html#extensionutils) be something like
```
import {Extension, gettext as _} from 'resource:///org/gnome/shell/extensions/extension.js';
```